google-cloud-dialogflow/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+### 0.4.0 / 2019-06-11
+
+* Add Intent::Message::Platform::GOOGLE_HANGOUTS
```

Comment:
I would list this after the model variant. This is not the most important change.

Reply:
Moved.